cmd/format-disk-cache.go

```diff
@@ -316,13 +302,30 @@ func cacheDrivesUnformatted(drives []string) bool {
 // Then validate the format for all drives in the cache to ensure order
 // of cache drives has not changed.
 func loadAndValidateCacheFormat(drives []string) (formats []*formatCacheV1, err error) {
+	driveDownCount := 0
+	for _, drive := range drives {
+		fi, err := os.Stat(drive)
+		if err != nil {
+			driveDownCount++
+			continue
+		}
+		if !fi.IsDir() {
+			return nil, fmt.Errorf("Cache drive %s is not a directory", drive)
+		}
+	}
+	if driveDownCount == len(drives) {
+		return nil, fmt.Errorf("Cache drives not available: %s", strings.Join(drives, ","))
+	}
```

Comment:
in the case when drive is not exist yet, loadAndValidateCacheFormat just returns "Cache drives not available" instead of initializing the cache